src/PythonMigrationViewExtension/GraphPythonDependencies.cs

```diff
@@ -9,10 +11,21 @@ namespace Dynamo.PythonMigration
     public class GraphPythonDependencies
     {
         private ViewLoadedParams ViewLoaded { get; set; }
+        public List<PythonNodeBase> GraphPythonNodes { get; set; }
 
         internal GraphPythonDependencies(ViewLoadedParams viewLoadedParams)
```

Comment:
if we don't need the entire `ViewLoadedParams`, let's only ask for the actual data we need, the `CurrentWorkspaceModel` or simply a list of `Nodes` for example ?